packages/components/src/custom-gradient-picker/style.scss

```diff
@@ -102,3 +97,18 @@ $components-custom-gradient-picker__padding: 6px; // 36px container, 24px handle
 		}
 	}
 }
+
+// All these styles should be made generic and changed on the inputs for all components.
+.components-custom-gradient-picker {
+	.components-select-control__input,
+	.components-input-control__input {
+		height: 40px !important;
+	}
+	.components-input-control__label {
+		line-height: 1;
+		padding-bottom: $grid-unit-10 !important;
+	}
+	label {
+		text-transform: uppercase;
```

Comment:
The metrics as I've extracted them from the Figma also make this inspector "Section Heading" 11px font-size with a font-weight of `500`.